app/packages/forms/views/form_details.jade

```diff
@@ -6,6 +6,7 @@ template(name="form_details")
         id=survey.id
         classNames='small'
         form=form
+        active=active
         showEdit=true)
```

Comment:
I think if we pass `survey` to the `form_header` template, this will work without needing to create a new helper in `form_details`. The global helper `surveyIsActive` will check the active state of the passed `survey` Parse obj.